app/repositories/SessionRepository.scala

```diff
@@ -21,26 +21,26 @@ import org.mongodb.scala.model.Filters.equal
 import org.mongodb.scala.model.Sorts.ascending
 import org.mongodb.scala.model.{IndexModel, IndexOptions, ReplaceOptions}
 import play.api.Configuration
-import play.api.libs.json.{Format, JsValue, Json, OFormat}
+import play.api.libs.json.{Format, JsValue, Json, OFormat, Reads, Writes, __}
 import uk.gov.hmrc.http.cache.client.CacheMap
 import uk.gov.hmrc.mongo.MongoComponent
 import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
 import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
 
-import java.time.{LocalDateTime, ZoneOffset}
+import java.time.{Instant, LocalDateTime, ZoneOffset}
 import java.util.concurrent.TimeUnit
 import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 case class DatedCacheMap(
   id: String,
   data: Map[String, JsValue],
-  lastUpdated: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC)
+  lastUpdated: Instant = Instant.now()
 )
 
 object DatedCacheMap {
-  implicit val dateFormat: Format[LocalDateTime] = MongoJavatimeFormats.localDateTimeFormat
```

Comment:
hmrc-mongo has removed LocalDateTime support and I decided to migrate over to instant from implementation of format method doesnt seems to be any difference so should be safe